### PR TITLE
Hiseq workflow type

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <skipIllumina>true</skipIllumina>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <jackson.version>2.9.9</jackson.version>
+    <jackson.version>2.9.9.2</jackson.version>
```
Comment (callunity) [inline pom.xml]: Will we be affected by this? https://github.com/FasterXML/jackson-databind/issues/2395

Comment (apmasell) [reply pom.xml]: Probably. I will rollback.

**Comment by callunity** [comment]
Failing test-- expected workflowType null, got workflowType None

**Comment by apmasell** [comment]
Yeah. Will look at it.

**Pull request merged**